certified-connectors/Powell Teams/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Powell.Teams.Web.PublicAPI",
+    "description": "Powell Teams API endpoints documentation.",
```

Comment:
@PowellSoftware This description is not allowed, Can you please provide a detailed explanation of your connector.

Documentation - [Description](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)